src/utilities/embeddable-utils.ts

```diff
@@ -61,7 +61,7 @@ export const getAnswerWithMetadata = (
     interactiveState: unknown,
     embeddable: {ref_id: string, authored_state?: string | null},
     oldAnswerMeta?: IExportableAnswerMetadata): IExportableAnswerMetadata => {
-
+      console.log(oldAnswerMeta);
```

Comment:
This should be removed or the `DEBUG_` system should be used to make it conditional.